core/capabilities/fakes/cron_trigger.go

```diff
@@ -0,0 +1,118 @@
+package fakes
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/triggers/cron"
+	crontriggermock "github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/triggers/cron/cron_triggermock"
+	sdkpb "github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2/pb"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/sdk/v2/testutils"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+type cronCapability struct {
+	doneCh chan bool
+	stopCh chan bool
+	lggr   logger.Logger
+	*testutils.CapabilityWrapper
+}
+
+func (c *cronCapability) RegisterTrigger(ctx context.Context, request capabilities.TriggerRegistrationRequest) (<-chan capabilities.TriggerResponse, error) {
+	ch := make(chan capabilities.TriggerResponse, 1)
+
+	go func() {
+		defer func() {
+			c.lggr.Debug("shutting down cron capability")
+			close(ch)
+			close(c.doneCh)
+		}()
+
+		for {
+			select {
+			case <-time.After(3 * time.Second):
```

Comment:
Why not support a real cron capability? It's so tiny that it could just be copied here IMO.